Cognite.Jetfire.Api/Credentials.cs

```diff
@@ -4,13 +4,16 @@
 using System;
 using System.Collections.Generic;
 using System.Net.Http;
+using System.Threading.Tasks;
+using Cognite.Extractor.Utils;
```

Comment:
Not ideal that you have to import all of the utils just to use the authenticator. Maybe we should move the authenticator to a separate package in the utils. I guess the only other dependency is Config?

Reply:
The config parts of the utils are not used. I considered rewriting the manifest parsing to use the config stuff, which would have given us some nice env var substitution for free, but I did not see a way of doing that without either braking the existing manifest schema or ending up with a very inconsistent schema.